### PR TITLE
Added Festive Moogles

### PR DESCRIPTION
**Pull request by kaincenteno**
Added scripts for Festive Moogles on each starter city.

### CODE REVIEW

**Comment by TeoTwawki** [comment]
this is wrong. This only happens if other conditions are met - you have no conditions here.

these npc's also have ***multiple events*** most of which are trade events (mog pells of various colors).

**Comment by kaincenteno** [comment]
@TeoTwawki  sure :cat:   i'll work on it today/tomorrow

**Comment by kaincenteno** [comment]
@TeoTwawki  changes made:
* added python script for GM to give items
* changed festive moogle to use variables, added cs if no variable assigned.
* commented trade for festive moogles for someone else to tackle it 😸

File: scripts/zones/Port_Bastok/npcs/Festive_Moogle.lua [hunk 1]
```diff
@@ -5,17 +5,33 @@
...
-        [2] = function(player) return (player:getCharVar("festiveMoogleMoogleCap")) end,
-        [3] = function(player) return (player:getCharVar("festiveMoogleMoogleRod")) end,
-        [4] = function(player) return (player:getCharVar("festiveMoogleHarpsichord")) end,
-        [5] = function(player) return (player:getCharVar("festiveMooglestuffedChocobo")) end,
-        [6] = function(player) return (player:getCharVar("festiveMoogleTidalTalisman")) end,
-        [7] = function(player) return (player:getCharVar("festiveMoogleDestrierBeret")) end,
-        [8] = function(player) return (player:getCharVar("festiveMoogleChocoboShurt")) end,
-    }
+function onTrade(player,npc,trade)
+    --TODO: trade of pellet for prize
```
Comment (TeoTwawki) [inline scripts/zones/Port_Bastok/npcs/Festive_Moogle.lua]: its called a "pell". not pellet.

Green Mog Pell
Red Mog Pell
Silver Mog Pell
Gold Mog Pell

etc etc

File: tools/festive_moogle/give_items.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+import mysql.connector as mariadb
+
+# Change with correct info ##
+sqlusername = "darkstar"    #
+sqlpassword = "password"    #
+sqldatabase = "dspdb"       #
+#############################
+
+
```
Comment (TeoTwawki) [inline tools/festive_moogle/give_items.py]: why would you even do this? its not hard to create char vars

File: scripts/zones/Windurst_Walls/npcs/Festive_Moogle.lua [hunk 3]
```diff
@@ -5,30 +5,57 @@
...
+    --TODO: trade of pellet for prize
 end
 
 function onTrigger(player,npc)
-    player:startEvent(503, unpack(stock))
+    local festiveItems = {getFestiveItems(player)}
+    if festiveItems[1] then
+        player:startEvent(380, unpack(festiveItems))
+    else
+        player:startEvent(381)
```
Comment (TeoTwawki) [inline scripts/zones/Windurst_Walls/npcs/Festive_Moogle.lua]: you've used the same csid at all 3 npc's. They do not use the same ID. the correct default event for the one in windurst is 504.

File: tools/festive_moogle/give_items.py (hunk 2 above)
Comment (kaincenteno) [reply tools/festive_moogle/give_items.py]: trying to be nice?
/shrug

Comment (kaincenteno) [reply tools/festive_moogle/give_items.py]: I can erase it if thats the ask =)

File: scripts/zones/Windurst_Walls/npcs/Festive_Moogle.lua (hunk 3 above)
Comment (kaincenteno) [reply scripts/zones/Windurst_Walls/npcs/Festive_Moogle.lua]: my bad. i copy and pasted it after making the changes, will fix this =)

File: scripts/zones/Port_Bastok/npcs/Festive_Moogle.lua (hunk 1 above)
Comment (kaincenteno) [reply scripts/zones/Port_Bastok/npcs/Festive_Moogle.lua]: got it, will rename it to pells =)

File: tools/festive_moogle/give_items.py (hunk 2 above)
Comment (TeoTwawki) [reply tools/festive_moogle/give_items.py]: someone else can decide if project wants this. ***not it***

Comment (TeoTwawki) [reply tools/festive_moogle/give_items.py]: @whasf I thrust the python thingy onto you.

Comment (TeoTwawki) [reply tools/festive_moogle/give_items.py]: silence is acceptance merging

**Pull request merged**